datarepo-clienttests/src/main/java/testscripts/PodDelete.java

```diff
@@ -0,0 +1,72 @@
+package testscripts;
+
+import bio.terra.datarepo.api.RepositoryApi;
+import bio.terra.datarepo.client.ApiClient;
+import bio.terra.datarepo.model.BulkLoadArrayRequestModel;
+import bio.terra.datarepo.model.JobModel;
+import java.util.List;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import utils.BulkLoadUtils;
+import utils.DataRepoUtils;
+import utils.KubernetesClientUtils;
+
+public class PodDelete extends runner.TestScript {
+  private static final Logger logger = LoggerFactory.getLogger(ScalePodsToZero.class);
+
+  /** Public constructor so that this class can be instantiated via reflection. */
+  public PodDelete() {
+    super();
```

Comment:
Can you set the manipulatesKubernetes property here so that we get a good error message when trying to run against localhost, for example?

Reply:
Thanks for pointing me to that!